vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/src/main/java/com/vaadin/flow/component/charts/export/NodeRunner.java

```diff
@@ -0,0 +1,26 @@
+package com.vaadin.flow.component.charts.export;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.vaadin.flow.server.frontend.FrontendTools;
+import com.vaadin.flow.server.frontend.FrontendUtils;
+
+public class NodeRunner {
```

Comment:
I saw the PR and description and was like AWESOME! I saw the node dependency on runtime and was like.. okay.. that's unexpected.. this should be mentioned as requirement at least.

Reply:
That's going to be mentioned on the documentation, yes.

Reply:
Having a note inside `SVGGenerators` javadoc wouldn't hurt I think